src/components/calendar/Event.tsx

```diff
@@ -47,40 +47,43 @@ export const isClosedEvent = (
 ) => proposalBooking?.status === ProposalBookingStatus.CLOSED;
 
 export const getBookingTypeStyle = (
-  bookingType: ScheduledEventBookingType
+  bookingType: ScheduledEventBookingType,
+  status: ProposalBookingStatus
 ): CSSProperties => {
+  const opacity = isDraftEvent({ status }) ? '0.6' : '1';
+
   switch (bookingType) {
     case ScheduledEventBookingType.COMMISSIONING:
       return {
-        backgroundColor: '#93E1D8',
+        backgroundColor: `rgba(147, 225, 216, ${opacity})`,
```

Comment:
Can we want to use colors from the theme palette?

Reply:
I guess we can but it was set like this from the beginning and I just continue using the used colors.